crypto/src/main/scala/org/bitcoins/crypto/DERSignatureUtil.scala

```diff
@@ -37,12 +37,16 @@ sealed abstract class DERSignatureUtil {
       val thirdByteIs0x02 = bytes(2) == 0x02
       //this is the size of the r value in the signature
       val rSize = bytes(3)
-
-      //this 0x02 separates the r and s value )in the signature
-      val second0x02Exists = bytes(rSize + 4) == 0x02
-
-      firstByteIs0x30 && signatureLengthIsCorrect && thirdByteIs0x02 &&
-      second0x02Exists
+      val sOffset = rSize + 4
```

Comment:
What is the bug here? Just an `OutOfBoundsException` if an invalid DER encoded `ByteVector` is passed into this function?